backup/src/main/scala/batches/BackupConfiguration.scala

```diff
@@ -0,0 +1,40 @@
+package batches
+
+import javax.inject.Singleton
+
+import com.google.inject.ImplementedBy
+import com.gu.conf.ConfigurationFactory
+import gu.msnotifications.NotificationHubConnection
+
+
+@ImplementedBy(classOf[BackupConfigurationFile])
+trait BackupConfiguration {
```

Comment:
None, it's a remain of me trying to write unit tests (see previous comment). I'll make it a class next commit.